controllers/controllers.js

```diff
@@ -1,6 +1,34 @@
+const User = require("../models/User");
+const Foundation = require("../models/Foundation");
+var jwt = require("jsonwebtoken");
+const config = require("../config/index");
 Pet = require("../models/Pet");
```

Comment:
you need to add type of declaration (const, let, var)

Reply:
Add type of declaration in controllers variables